src/main/java/codeu/controller/TagServlet.java

```diff
@@ -167,24 +175,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       List<String> splitTags = Arrays.asList(tags.split(", "));
 
       for (String tag : splitTags) {
-        // Creation of a new Tag object
-        if (!tagStore.isTagTaken(tag)) { 
-          Tag newTag = new Tag(UUID.randomUUID(), conversation.getId(), tag.toLowerCase(), Instant.now());
-          tagStore.addTag(newTag);
-          conversation.addTag(newTag);
-        }
-        // User inputs a Tag that is already known to the application
-        else {
-          Tag oldTag = tagStore.getTag(tag);
-          oldTag.addConversation(conversation.getId());
-          tagStore.putTag(oldTag);
-          conversation.addTag(oldTag);
-        }
+        Tag newTag = new Tag(UUID.randomUUID(), conversation.getId(), tag.toLowerCase(), Instant.now());
+        tagStore.addTag(newTag);
```

Comment:
So are we now associated one `Tag` object with one `Conversation` object only?

Reply:
yup